packages/slider/src/mwc-slider-base.ts

```diff
@@ -226,6 +231,7 @@ export class SliderBase extends FormElement {
     };
   }
 
+  @eventOptions({capture: true, passive: true})
```

Comment:
Can you add a comment about why we need to call layout on mousedown and touchmove? Seems like that could be expensive because, isn't touchmove going to get fired a lot when a user is interacting with the slider? I'm not sure I understand the original problem this is fixing very well.

Reply:
meant to do touchstart

Reply:
fixed

Reply:
I think it still needs a comment.

Reply:
I still don't see a comment.

Reply:
weird

Reply:
thanks for the catch!